multimedia/templates/multimedia/base.html

```diff
@@ -0,0 +1 @@
+{% extends "base.html" %}
```

Comment:
Do I need to design this as well? (Since I'm the one in charge with the overall design...)